persistence/test/application_test.go

```diff
@@ -209,26 +190,25 @@ func newTestApp() (*typesGenesis.App, error) {
 		return nil, err
 	}
 
-	return &typesGenesis.App{
+	return &genesis.App{
 		Address:         operatorKey.Address(),
 		PublicKey:       operatorKey.Bytes(),
 		Paused:          false,
-		Status:          typesGenesis.DefaultStakeStatus,
-		Chains:          typesGenesis.DefaultChains,
+		Status:          genesis.DefaultStakeStatus,
+		Chains:          genesis.DefaultChains,
 		MaxRelays:       DefaultMaxRelays,
-		StakedTokens:    typesGenesis.DefaultStake,
+		StakedTokens:    genesis.DefaultStake,
 		PausedHeight:    DefaultPauseHeight,
 		UnstakingHeight: DefaultUnstakingHeight,
 		Output:          outputAddr,
 	}, nil
 }
 
+// DISCUSS_IN_THIS_COMMIT: We are only calling these functions and tests for apps, but need to
```

Comment:
Okay, is this a discussion?

Reply:
Sorry, this was mean to be a `TODO_IN_THIS_COMMIT`